### PR TITLE
[WiP] Fix ci

### PR DESCRIPTION
**Pull request by andrii-suse**

### CODE REVIEW

File: t/docker/lib/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+RUN zypper -n addrepo https://download.opensuse.org/repositories/openSUSE:/infrastructure/openSUSE_Leap_15.1/ openSUSE:infrastructure
+RUN zypper mr -p 97 openSUSE:infrastructure
+RUN zypper -n addrepo https://download.opensuse.org/repositories/server:/database:/postgresql/openSUSE_Leap_15.1/ server:database:postgresql
+RUN zypper mr -p 98 server:database:postgresql
+
+RUN zypper -n --gpg-auto-import-keys --no-gpg-checks refresh
+
+RUN zypper -n install postgresql10-ip4r postgresql postgresql-server
+
+RUN zypper -n install mirrorbrain mirrorbrain-scanner apache2-worker apache2-mod_asn apache2-mod_geoip apache2-mod_form mirrorbrain-tools
```
Comment (darix) [inline t/docker/lib/Dockerfile:30]: apache2-mod_maxminddb instead of mod_geoip

File: t/docker/lib/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
+RUN zypper -n install apache2-devel
+RUN zypper -n install python3-SQLObject
+RUN zypper -n install python3-FormEncode
```
Comment (darix) [inline t/docker/lib/Dockerfile:33]: add python3-geoip2 (the name is confusing it does maxminddb)

File: t/docker/lib/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,49 @@
...
+RUN zypper -n install python3-devel python3-cmdln python3-geoip2 python3-psycopg2
+
+RUN zypper -n addrepo https://download.opensuse.org/repositories/openSUSE:/infrastructure/openSUSE_Leap_15.1/ openSUSE:infrastructure
+RUN zypper mr -p 97 openSUSE:infrastructure
+RUN zypper -n addrepo https://download.opensuse.org/repositories/server:/database:/postgresql/openSUSE_Leap_15.1/ server:database:postgresql
+RUN zypper mr -p 98 server:database:postgresql
+
+RUN zypper -n --gpg-auto-import-keys --no-gpg-checks refresh
+
+RUN zypper -n install postgresql10-ip4r postgresql postgresql-server
```
Comment (darix) [inline t/docker/lib/Dockerfile:28]: just to make sure we dont run into the same problem as Frank we should probably: postgresql10-ip4r postgresql10 postgresql10-server

File: t/docker/lib/Dockerfile (hunk 2 above)
Comment (darix) [reply t/docker/lib/Dockerfile:33]: Oh I see you have that up higher already. maybe group those lines?

File: t/docker/lib/init/install-from-source.sh [hunk 4]
```diff
@@ -14,7 +14,7 @@ apxs -cia -lm mod_mirrorbrain.c
 
 ( 
 rm /usr/bin/mb
-rm /usr/lib64/python*/site-packages/mb/*
+rm -f /usr/lib64/python*/site-packages/mb/*
```
Comment (darix) [inline t/docker/lib/init/install-from-source.sh:17]: replace it with a "uninstall" of the mirrorbrain packages above.

File: t/docker/lib/Dockerfile (hunk 2 above)
Comment (andrii-suse) [reply t/docker/lib/Dockerfile:33]: yeah good idea - I was just appending with every run to reuse docker cache.
But in my understanding these will be installed anyway as dependency of mirrorbrain packages? (when that part is python3 ready)

Comment (darix) [reply t/docker/lib/Dockerfile:33]: I can only move the package to python3 once we are done here. :) chicken & egg problem

Comment (andrii-suse) [reply t/docker/lib/Dockerfile:33]: ok, then we can go with plain dockerfile and maybe will remove dependencies and use container from OBS

**Comment by andrii-suse** [comment]
Closing in favor of #8 because they are conflicting now

**Pull request closed without merging**